src/instrument.ts

```diff
@@ -45,12 +45,21 @@ interface SQLStatement {
   sql: string;
 }
 
-interface observabilityOptions {
+/*
+ * ObservabilityOptions defines the configuration with which
+ * startTrace may be invoked if set.
+ * @property [tracerProvider] is the injected TracerProvider to use,
+ * otherwise the global TracerProvider shall be used.
```

Comment:
```suggestion
 * ObservabilityOptions defines the configuration to be used for Spanner OpenTelemetry Traces.
 * @property [tracerProvider] Sets the TracerProvider to be used for traces,
 * Global TracerProvider will be used as a fallback.
```

Reply:
Done and thank you @surbhigarg92! Kindly help me re-run the bots.